api/src/pcapi/core/categories/subcategories_v2.py

```diff
@@ -173,6 +173,7 @@ class ExtraDataFieldEnum(Enum):
     SPEAKER = "speaker"
     STAGE_DIRECTOR = "stageDirector"
     VISA = "visa"
+    GTL_ID = "gtlId"
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏J'ai eu un doute, j'ai vérifié, ici on veut `gtl_id` comme dans le modèle `OfferExtraData`
C'est dommage qu'on ait un mix camelCase et snake_case dans le modèle effectivement :/

Reply:
ok je pense que c'est mieux par contre on est d'accord que c'est indépendant. la base de donné et ça? ou j'ai mal compris?

Reply:
Dans la base de donnée, on aura `gtl_id`
Du coup dans ta PR, pour la sandbox c'est bon parce que tu fais
```
if (blabla):
    extraData['gtl_id'] = 1111
``` 
Du coup c'est bon
Mais le reste du code qui utilise `ConditionalField` ne va pas marcher